.travis.yml

```diff
@@ -0,0 +1,38 @@
+sudo: required
+
+services:
+  - docker
+
+matrix:
+  fast_finish: true
+
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
```

Comment:
Why is the `-o Dpkg::Options::="--force-confold" --force-yes` needed when installing `docker-engine`?